src/CoreWCF.WebHttp/src/CoreWCF/Dispatcher/IDataContractSurrogate.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace CoreWCF.Dispatcher
+{
+    internal interface IDataContractSurrogate
+    {
```

Comment:
I can't find this used anywhere. There's also a new surrogate interface if there's any reference to this which should be used instead.